src/lib/tuya.ts

```diff
@@ -1684,8 +1693,8 @@ const tuyaModernExtend = {
         return tuyaModernExtend.dpEnumLookup({name: 'power_on_behavior', lookup: lookup, type: dataTypes.enum,
             expose: e.power_on_behavior(Object.keys(lookup)).withAccess(readOnly ? ea.STATE : ea.STATE_SET), ...args});
     },
-    tuyaLight(args?: modernExtend.LightArgs & {minBrightness?: boolean, switchType?: boolean}) {
-        args = {minBrightness: false, powerOnBehavior: false, switchType: false, ...args};
+    tuyaLight(args?: modernExtend.LightArgs & {minBrightness?: boolean, minBrightness2?: boolean, switchType?: boolean}) {
```

Comment:
```suggestion
    tuyaLight(args?: modernExtend.LightArgs & {minBrightness?: 'attribute' | 'command', switchType?: boolean}) {
```

Reply:
I added also "none" for devices that cannot set minimum brightness. Not sure if that is right.